examples/fungible/src/lib.rs

```diff
@@ -72,6 +70,18 @@ where
     }
 }
 
+#[derive(Clone, Debug, Deserialize, Eq, Ord, PartialEq, PartialOrd, Serialize)]
+pub struct FungibleInitial {
```

Comment:
Note that we haven't prefixed the other definitions with `Fungible` and the builder is called `InitialStateBuilder`, so a logical choice here is `InitialState`.

Reply:
Also `initial` is not a noun, so when one writes `let initial = FungibleInitial { .. }` it doesn't work as nicely as `let state = InitialState { }` or `let accounts = InitialAccounts { }`.

Reply:
Ok for `let state = InitialState { ... }` which sounds good to me.